### PR TITLE
Use findorad instead of abci and tendermint.

### PR DESCRIPTION
**Pull request by tiannian**

### CODE REVIEW

File: docs/general/run-a-testnet-validator-node.md [hunk 1]
```diff
@@ -162,18 +154,25 @@ perl -pi -e \
 
 ```shell
 # Start your validator process
-abci_validator_node \
-    --ledger-dir="${ROOT_DIR}/abci" \
-    --tendermint-node-key-config-path="${HOME}/.tendermint/config/priv_validator_key.json" \
+docker run -d \
+    -v $HOME/.tendermint:/root/.tendermint \
```
Comment (ktmlm) [inline docs/general/run-a-testnet-validator-node.md:158]: `root` ?

File: docs/general/run-a-testnet-validator-node.md [hunk 2]
```diff
@@ -70,27 +62,27 @@ rm -rf ~/.tendermint
...
+docker run --rm -v $HOME/.tendermint:/root/.tendermint public.ecr.aws/k6m5b6e2/release/findorad init
 
 # Create ledger data directory, for example
 rm -rf ${ROOT_DIR}
 mkdir -p ${ROOT_DIR}/abci ${ROOT_DIR}/tendermint
 ```
 
 > **Tips**:
-> - If you encounter a security issue error when trying to run `tendermint init`, you may need to manually approve its security priveliges in you OS first. Then re-run the `tendermint init` command again.
+> - If you encounter a security issue error when trying to run `findorad init`, you may need to manually approve its security priveliges in you OS first. Then re-run the `findorad init` command again.
```
Comment (ktmlm) [inline docs/general/run-a-testnet-validator-node.md:73]: change to docker style ?

File: docs/general/run-a-testnet-validator-node.md [hunk 3]
```diff
@@ -46,9 +38,9 @@ Download the following files:
 > **Tips**:
 > - You can (optionally) run a Linux node via `Windows Subsystem for Linux`
 > - Check that binaries have executable permissions set correctly
->     - ex) `chmod +x tendermint abci_validator_node fns`
+>     - ex) `chmod +x findorad fns`
 > - Check that binary files are placed into one of your `PATH` directories
->     - ex) `mv tendermint abci_validator_node fns /usr/local/bin/`
+>     - ex) `mv findorad fns /usr/local/bin/`
```
Comment (ktmlm) [inline docs/general/run-a-testnet-validator-node.md:43]: change to docker style ?

remove findorad here ?

File: docs/general/node_init.sh [hunk 4]
```diff
@@ -21,16 +21,15 @@ check_env() {
 set_binaries() {
     OS=$1
 
-    wget -T 10 https://github.com/FindoraNetwork/testnet-downloads/releases/download/${OS}/tendermint || exit 1
-    wget -T 10 https://github.com/FindoraNetwork/testnet-downloads/releases/download/${OS}/abci_validator_node || exit 1
+    wget -T 10 https://github.com/FindoraNetwork/testnet-downloads/releases/download/${OS}/findorad || exit 1
```
Comment (ktmlm) [inline docs/general/node_init.sh:24]: should here be changed to `pull docker image` ?

File: docs/general/node_init.sh [hunk 5]
```diff
@@ -21,16 +21,15 @@ check_env() {
...
     wget -T 10 https://github.com/FindoraNetwork/testnet-downloads/releases/download/${OS}/fns || exit 1
     wget -T 10 https://github.com/FindoraNetwork/testnet-downloads/releases/download/${OS}/stt || exit 1
 
     new_path=/tmp/findora_testnet_bin
 
     rm -rf $new_path 2>/dev/null
     mkdir -p $new_path || exit 1
-    mv tendermint abci_validator_node fns stt $new_path || exit 1
+    mv findorad fns stt $new_path || exit 1
```
Comment (ktmlm) [inline docs/general/node_init.sh:32]: change to docker style ?

File: docs/general/node_init.sh [hunk 6]
```diff
@@ -80,7 +79,7 @@ fi
 pkill -9 tendermint
 rm -rf ~/.tendermint 2>/dev/null
 
-tendermint init || exit 1
+findorad init || exit 1
```
Comment (ktmlm) [inline docs/general/node_init.sh:82]: change to docker style ?

File: docs/general/node_init.sh [hunk 7]
```diff
@@ -94,20 +93,18 @@ perl -pi -e "s#(persistent_peers = )\".*\"#\$1\"${SENTRY}\"#" ~/.tendermint/conf
 # Run locale node #
 ###################
 
-pkill -9 abci_validator_node
+pkill -9 findorad
 mkdir -p ${ROOT_DIR}/{abci,tendermint}
 
 cd ${ROOT_DIR}/abci
-nohup abci_validator_node \
+nohup findorad \
```
Comment (ktmlm) [inline docs/general/node_init.sh:100]: change to docker style ?

File: docs/general/node_init.sh [hunk 8]
```diff
@@ -94,20 +93,18 @@ perl -pi -e "s#(persistent_peers = )\".*\"#\$1\"${SENTRY}\"#" ~/.tendermint/conf
 # Run locale node #
 ###################
 
-pkill -9 abci_validator_node
+pkill -9 findorad
```
Comment (ktmlm) [inline docs/general/node_init.sh:96]: `docker stop` ?

**Review by ktmlm — CHANGES_REQUESTED** [review]
**nice work, but some details need to be optimized @tiannian @simonjiao**

**Comment by tiannian** [comment]
ok, i will fix these.

File: docs/general/run-a-testnet-validator-node.md (hunk 1 above)
Comment (tiannian) [reply docs/general/run-a-testnet-validator-node.md:158]: In this docker image, default user is root.

**Pull request merged**